CONTRIBUTING.md

```diff
@@ -0,0 +1,316 @@
+If it is not urgent, the common courtesy ping rate is one week. Remember that you’re asking for valuable time from other professional developers.
+
+Note that anyone is welcome to review and give feedback on a change, but only people with commit access to the repository can approve it.
+
+### Testing
+
+Developers are required to create test cases for any bugs fixed and any new features added, and to contribute them along with the changes.
+
+- All feature and regression test cases are added to the appropriate test directory—for example, the `swift/test` directory.
+- Write test cases at the abstraction level nearest to the actual feature.
+For example, if it’s a Swift language feature, write it in Swift; if it’s a SIL optimization, write it in SIL.
+- Reduce test cases as much as possible, especially for regressions.
+It’s unacceptable to place an entire failing program into `swift/test` because this slows down testing for all developers.
+Please keep them short.
+
+### Quality
+
+People depend on Swift to create their production software.
+This means that a bug in Swift could cause bugs in thousands, even millions of developers’ products.
+Because of this, the Swift project maintains a high bar for quality.
+The minimum quality standards that any change must satisfy before being committed to the main development branch include:
+
+1. Code must compile without errors or warnings on at least one platform.
+2. Bug fixes and new features must include a test case to pinpoint any future regressions, or include a justification for why a test case would be impractical.
+3. Code must pass the appropriate test suites—for example, the `swift/test` and `swift/validation-test` test suites in the Swift compiler.
+
+Additionally, the committer is responsible for addressing any problems found in the future that the change may cause.
+This responsibility means that you may need to update your change in order to:
+
+- Ensure the code compiles cleanly on all primary platforms.
+- Fix any correctness regressions found in other test suites.
+- Fix any major performance regressions.
+- Fix any performance or correctness regressions in the downstream Swift tools.
+- Fix any performance or correctness regressions that result in customer code that uses Swift.
+- Address any bugs that appear in the bug tracker as a result from your change.
+
+We prefer that these issues be handled before submission, but we understand that it isn’t possible to test all of this for every submission.
+Our continuous integration (CI) infrastructure normally finds these problems.
+We recommend watching the CI infrastructure throughout the next day to look for regressions.
+The CI infrastructure will directly email you if a group of commits that included yours caused a failure.
+You are expected to check those messages to see whether they are your fault and, if so, fix the breakage.
```

Comment:
I don’t think this paragraph contributes to anything. CI testing is required for a PR to be merged and I don’t think it’s reasonable for anyone to *watch the CI infrastrucute* throughout the next day. Also, the CI infrastructure doesn’t send failure emails. Let’s just remove it.

Reply:
Should be gone now with the latest commit